examples/examples-cpu/snowflake/snowflake-dask.ipynb

```diff
@@ -0,0 +1,388 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can call `compute()` to execute the function and see the output (in this case a Pandas dataframe)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "type(out.compute())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's load more days using Dask! First we want to pull a range of dates where know data exists. We can run a quick Snowflake query for that"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "date_query = \"\"\"\n",
+    "SELECT\n",
+    "    DISTINCT(DATE(pickup_datetime)) as date \n",
+    "FROM taxi_yellow\n",
+    "WHERE\n",
+    "    pickup_datetime BETWEEN '2019-01-01' and '2019-01-31'\n",
+    "\"\"\"\n",
+    "dates_df = conn.cursor().execute(date_query).fetch_pandas_all()\n",
+    "dates = dates_df['DATE'].tolist()\n",
+    "dates[:5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then, we build up a list of delayed objects that call the `load()` function we created"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "delayed_obs = [load(conn_info, query, day) for day in dates]\n",
+    "delayed_obs[:5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, create a Dask Dataframe!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ddf = dd.from_delayed(delayed_obs)\n",
+    "ddf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that the above command ran pretty quickly. This is because Dask only executes the task graph when you perform certain actions, such as writing a file or getting the `len` of the DataFrame"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "len(ddf)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<br>\n",
+    "\n",
+    "`repartition()` to introduce more parallelism. This helps downstream processes execute faster by splitting the work across more cores."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ddf = ddf.repartition(npartitions=100)\n",
+    "ddf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "len(ddf)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<br>\n",
+    "The cell below will execute the Snowflake queries across the cluster and compute the length and size of each partition"
```

Comment:
this language was a little  confusing to me. The code below it prints the number of rows and the memory usage of *the entire dataframe*, not the per-partition number of rows and memory footprint. I think this sentence could be confusing and make a beginner think that all partitions are the same size.

I also think you should use the language "number of rows" instead of "length". It's a convenience that `len(df)` prints the number of rows, but I personally think talking about the "length" of a data frame is kind of unintuitive, and that "number of rows" and "number of columns" is clearer.